infrastructure/permissions.tf

```diff
@@ -50,12 +50,12 @@ resource "aws_iam_policy_attachment" "fleet_role" {
 }
 
 resource "aws_iam_instance_profile" "data_refinery_instance_profile" {
-  name  = "data-refinery-instance-profile-${var.user}-${var.stage}"
+  name = "data-refinery-instance-profile-${var.user}-${var.stage}"
   role = "${aws_iam_role.data_refinery_instance.name}"
 }
 
 resource "aws_iam_policy" "s3_access_policy" {
-  name = "data-refinery-s3-access-policy-${var.user}-${var.stage}"
+  name        = "data-refinery-s3-access-policy-${var.user}-${var.stage}"
```

Comment:
Waiiiit noo, I hate this style and want to try to get more standard around doing it without unnecessary spaces in the middle of our lines. There's a few places where we do this cause the code got copied from elsewhere without getting fixed but we should fix that stuff not change everything else.